src/main/java/cloud/orbit/actors/extensions/metrics/dropwizard/OrbitActorExtension.java

```diff
@@ -61,19 +58,35 @@
 
     public static final String ACTOR_METRICS_PIPELINE_NAME = "actor-metrics-pipeline";
 
-    //Timer, per actor instance
-    Map<String, Timer.Context> actorActivationTimers = new ConcurrentHashMap<>();
-    Map<String, Timer.Context> actorLifeSpanTimers = new ConcurrentHashMap<>();
-    Map<String, Timer.Context> actorDeactivationTimers = new ConcurrentHashMap<>();
+    private Map<String, Timer.Context> actorActivationTimers = new ConcurrentHashMap<>();
+    private Map<String, Timer.Context> actorLifeSpanTimers = new ConcurrentHashMap<>();
+    private Map<String, Timer.Context> actorDeactivationTimers = new ConcurrentHashMap<>();
+
+    private MetricRegistry metricRegistry;
 
     public OrbitActorExtension()
```

Comment:
I know you didn't create this, but seeing as this is a big change is there any chance we could rename this class?
OrbitActorExtension is a really generic name, I feel like "Metrics" should be in there somewhere.